CRM/Event/DAO/Participant.php

```diff
@@ -635,6 +645,23 @@ public static function &fields() {
           ],
           'add' => '4.7',
         ],
+        'created_id' => [
+          'name' => 'created_id',
+          'type' => CRM_Utils_Type::T_INT,
+          'title' => ts('Contact responsible for registering this participant'),
+          'description' => ts('FK to contact table.'),
```

Comment:
This is incorrect and doesn't match the xml. Please regenerate this file instead of updating it by hand. Command is `bin/setup.sh -g`

Reply:
Done, not sure how I missed that!